resources/language/resource.language.nl_nl/strings.po

```diff
@@ -1042,3 +1042,7 @@ msgstr "Geen on demand stream beschikbaar voor {title}."
 msgctxt "#30987"
 msgid "No stream found, please try again!"
 msgstr "Geen stream gevonden, gelieve opnieuw te proberen!"
+
+msgctxt "#30988"
+msgid "InputStream Adaptive is not installed/enabled, pause/play and timeshift of live streams will not work"
```

Comment:
We need to update this as well.
```suggestion
msgid "InputStream Adaptive is not installed, therefore pause and timeshift of live streams may not work"
```

Reply:
This is not resolved, this is the English `msgid` in the Dutch translation. If this is not identical, it will not be translated to Dutch.